src/Gluon.Client/package.json

```diff
@@ -1,13 +1,9 @@
 {
-  "name": "gluon",
-  "version": "2.0.0",
+  "name": "gluon-client",
+  "version": "4.0.0",
   "description": "TypeScript to FSharp connector",
-  "main": "Gluon.ts",
-  "devDependencies": {
-    "rimraf": "2.5.1",
-    "typescript": "2.1.4",
-    "uglify-js": "2.6.1"
-  },
+  "main": "./dist/Gluon.js",
+  "typings": "./dist/Gluon.d.ts",
```

Comment:
This provides consumers of the library enough information to pull in both the library and its typings without requiring additional configuration in tsconfig.json.